frontend/src/i18n/locales/de/glossary.json

```diff
@@ -97,6 +97,8 @@
   "Uncordon": "Planungssperre aufheben",
   "Cordon": "Planungssperre setzen",
   "Drain": "Entleeren",
+  "Drop Node Shell": "Drop Node Shell",
+  "Drop Node Shell is not support on this operatingSystem: {{ device }}": "Drop Node Shell is not support on this operatingSystem: {{ device }}",
```

Comment:
All translations except the English are supposed to be empty. So we can at a later point fetch them and translate them. I know this generates a warning in the dev console, but that's okay.

Reply:
This has been addressed. Resolving.